com.ibm.streamsx.topology/com.ibm.streamsx.topology.functional.python/pyfunction.pm

```diff
@@ -22,18 +22,23 @@ sub splpy_tuplestyle{
 
  my $attr =  $port->getAttributeAt(0);
  my $pystyle = 'unk';
- if ($attr->getName() eq '__spl_po') {
+ my $itupleType = $port->getSPLTupleType();
+ my $numattrs = $port->getNumberOfAttributes();
+ if (($numattrs == 1) && ($attr->getName() eq '__spl_po')) {
     $pystyle = 'pickle';
- } elsif ($attr->getName() eq 'string') {
+ } elsif (($numattrs == 1) && ($attr->getName() eq 'string')) {
     $pystyle = 'string';
- } elsif ($attr->getName() eq 'jsonString') {
+ } elsif (($numattrs == 1) && ($attr->getName() eq 'jsonString')) {
     $pystyle = 'json';
  }
+ else {
+    $pystyle = 'spltuple';
```

Comment:
We will support multiple styles for a SPL tuple so maybe the name could reflect the type of the style? 'dict' ?